### DISCUSSION TITLE
license question with raisimgymtorch

### DISCUSSION THREAD

**Discussion by sbs0323** [raisimTech/raisimLib#220] [Q&A]
Hello prof.hwangbo
I  need your help plesase.
i got activation raisim key and put it correctly. raisim default examples are going well.
However i cannot solve certain license problem.. during using raisimgymtorch

-problem code part- maybe
![image](https://user-images.githubusercontent.com/48010295/139828393-a9657969-4eb5-48ce-8452-13ef3c07e057.png)
  
-my print out results-
start3.5  
start3  
[2021:11:02:19:05:26 C:\Users\jemin\Documents\git\raisim\include_internal\license.hpp:23] [1;31mCannot find the license file         (       /                             [0m  

i think license.hpp file [23 row] has some information. 
can you guess what this problem is?

**Comment by jhwangbo** [comment]
If you saved your license file in .raisim folder as shown in the doc, you can delete setActivationKey line

**Reply by sbs0323** [reply]
however, error seems occur in this line

world_ = std::make_unique<raisim::World>(); 

if there is no setActivationKey line, nothing was changed.

**Reply by jhwangbo** [reply]
Please pull again and try
